libkodimote/kodi.cpp

```diff
@@ -168,6 +168,8 @@ Kodi::Kodi(QObject *parent) :
     qmlRegisterType<KodiHost>(qmlUri, 1, 0, "KodiHost");
     qmlRegisterType<KodiDiscovery>(qmlUri, 1, 0, "KodiDiscovery");
 
+    qmlRegisterUncreatableType<Settings>("Kodi", 1, 0, "Settings", "Cannot create multiple settings. Use the \"settings\" context property intead");
```

Comment:
Instead of `"Kodi"` you should have used `qmlUri`, as for Sailfish I need to use `harbour-kodimote` (matching the package name). (will fix in my branch)